src/command/build.rs

```diff
@@ -54,6 +54,8 @@ pub enum Target {
     /// in a browser but pollutes the global namespace and must be manually
     /// instantiated.
     NoModules,
+    /// Correspond to `--target all` where the output is all other targets
+    All,
```

Comment:
given @Pauan's comment and the likelihood that folks will want to customize the types of targets they want... i could imagine that instead of an `All` type, you have a new type that is a `Targets: Vec<Target>`- and for just Node, the Targets would by `Vec<Nodejs>`, and all would be `Vec<Web, Nodejs`, Modules>